src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -30,15 +30,22 @@ uiModules.get('apps/management')
 
   // Configure the new index pattern we're going to create.
   this.formValues = {
-    id: $routeParams.id ? decodeURIComponent($routeParams.id) : undefined,
     name: config.get('indexPattern:placeholder'),
```

Comment:
Why not follow the same pattern that was there?
`name: $routeParams.id ? '' : config.get('indexPattern:placeholder')`